src/common/utils/hooks/useAssetHub.ts

```diff
@@ -30,37 +30,18 @@ export const useAssetHub = () => {
     return { transferDotFee, dotED, totalDotFee };
   };
 
-  const getGiftFee: GetAssetHubGiftFee = async ({ chainId, assetId, transferAmmount, address }) => {
-    const { transferDotFee, dotED, totalDotFee } = await getTransferFee({
-      chainId,
-      assetId,
-      transferAmmount,
-      address,
-    });
-
-    // Add more fee if it's a gift
-    const giftDotFee = totalDotFee + transferDotFee + dotED;
-
-    return { totalDotFee: giftDotFee };
-  };
-
```

Comment:
We need this part to add fee for gifts here or in `sendGifts`
But if we change the part with ED AH fee calculation then we can improve this part and have a gift fee simply as `transferDotFee*2`